kubernetes/RELEASES.md

```diff
@@ -0,0 +1,127 @@
+# Deploy to Maven release process
+
+This document describes how to configure and use the [Maven release
+plugin](http://maven.apache.org/maven-release/maven-release-plugin) to publish
+to [Sonatype](http://central.sonatype.org/).
+
+Releases are done on an as-needed basis, and this doc applies only to
+[OWNERS](https://github.com/kubernetes-client/java/blob/master/OWNERS).
+
+This does _not_ describe the process of cherry-picking changes onto release
+branches.
+
+## One time setup
+
+You will need to have the following in place:
+
+1. SSH keys for an account capable of pushing commits & tags to
+   https://github.com/kubernetes-client/java. These will be used by the release
+   plugin to push an updated pom.xml along with a tag corresponding to the
+   release being performed. If you don't have these keys, follow [this
+   guide](https://help.github.com/articles/connecting-to-github-with-ssh/).
+
+2. [A Sonatype JIRA
+   account](https://issues.sonatype.org/secure/Signup!default.jspa) that's been
+   authorized to publish to `io.kubernetes:client-java`. With your credentials
+   in-hand, place these in your `settings.xml` (typically
+   `~/.m2/settings.xml`) config file. An example config is:
+   ```xml
+<settings>
+  <servers>
+    <server>
+      <id>ossrh</id>
+      <username>your-jira-id</username>
+      <password>your-jira-pwd</password>
```

Comment:
I haven't found a way, but will look - keep in mind this file lives outside the source directory, and instead in `~/.m2/settings.xml`